dockerfile_scripts/install_deepspeed.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEBIAN_FRONTEND=noninteractive apt-get install -y pdsh libaio-dev
+if [ "$DEEPSPEED_PIP" ]; then
+    python -m  pip install ninja psutil packaging attrdict
+    DS_BUILD_OPS=1 python -m  pip install $DEEPSPEED_PIP
```

Comment:
while we do still have `python -m pip install` in a couple places, it's not necessary

```suggestion
    DS_BUILD_OPS=1 pip install $DEEPSPEED_PIP
```

Reply:
should i standardize all of the pip install calls in this PR?  what's best practices here?

Reply:
I wouldn't worry about changing other stuff - though like updating the Ubuntu version, we should probably do that some other time. I suspect this was necessary at some point because of base systems that had conflicting Python versions like 2/3, Ubuntu/Anaconda, etc. but it shouldn't be now.